### ISSUE TITLE
Not unblocking

### ISSUE THREAD

**Issue by drtiwarir** [Seji64/SniDust#31]
I did configure it on a VPS in India, but all the streaming websites I visited give error ' you seem to be using a unblocker or proxy'
Any solution to this ?
When I use paid Smart DNS proxy, all websites are working.
Please fix it 
Regards

**Comment by drtiwarir** [comment]
Hi,
I did some search and feel that when we are using SniDust, it works like a proxy, all data flows through VPS. Now almost all streaming websites have blocked data centre IP addresses.
I did check how paid DNS service works, they use data centre IP address, but the data doesn't flow through VPS (that's why they are cheap)
In your project, I think the problem is with the External IP, what paid service providers are doing is using a domestic (home broadband) IP address, which can't be blocked by any streaming service provider, this IP address is just used to establish connection, as soon as the stream starts, it goes directly to the client's IP address.
Domestic IP address is just used to spoof location, it does not receive and data.
If you can play with IP tables and try ? 
I am may be wrong, my apologies in advance.
Please let me know your thoughts.

**Comment by drtiwarir** [comment]
How about using MITM proxy ? instead of Sniproxy

**Comment by Seji64** [comment]
No idea how MITM could help here. That is something completely different. You are comparing pears with apples here.

**Comment by drtiwarir** [comment]
Sorry bro 
Just had a thought 
I agree with you, my apologies

**Comment by drtiwarir** [comment]
I tried a paid dns proxy and found that the initial handshake or resolution is done through a mobile proxy (no data centre IP address)
I did check about mobile proxies on github and found an interesting one - https://github.com/proxidize/proxidize-android
Can you try sending the requests through mobile proxy ? instead of showing VPC's IP address (EXTERNAL_IP=10.111.123.8) ?

**Comment by Seji64** [comment]
That is [IMHO](https://de.wiktionary.org/wiki/IMHO) out of scope for this project.
Regardless, I don't think it would do any good to implement it because even then the proxy has to run somewhere. And I think it is quite unrealistic that you can rent android phones somewhere to install a SOCKS proxy on it. In addition, they need a SIM card and a mobile phone contract (with a very very large data plan).
